config/main.py

```diff
@@ -4513,6 +4513,13 @@ def ip(ctx):
     """Set IP interface attributes"""
     pass
 
+def validate_vlan_format(text):
+    pattern = re.compile(r'^Vlan([1-9]\d{0,2}|[1-3]\d{3}|40[0-8][0-9]|409[0-4])$')
```

Comment:
This is a pretty complicated regex, make sure to test this with multiple different valid (and invalid) strings if you haven't already.
I recommend https://regexr.com and copying some vlan strings to validate this expression works as expected.

Reply:
Thanks :-)